src/assets/toolkit/scripts/home-animation.js

```diff
@@ -0,0 +1,300 @@
+/**
+ * GSAP is powerful and performant, but its organization is a little old-school.
+ * This imports a bunch of functions to the global scope.
+ *
+ * See: https://github.com/greensock/GreenSock-JS/issues/71
+ */
+
+import TweenLite from 'gsap/src/uncompressed/TweenLite';
+import TimelineMax from 'gsap/src/uncompressed/TimelineMax';
+import EasePack from 'gsap/src/uncompressed/easing/EasePack';
+import CSSPlugin from 'gsap/src/uncompressed/plugins/CSSPlugin';
+import AttrPlugin from 'gsap/src/uncompressed/plugins/AttrPlugin';
```

Comment:
That is unfortunate. The following does work, but because it uses the main library (`TweenMax`) the resulting file is 1300 lines longer.

```js
import {
  default as TweenLite,
  TimelineMax,
  EasePack,
  CSSPlugin,
  AttrPlugin
} from 'gsap';
```